src/modules/Home/components/MainSlider/components/MainSlide/MainSlide.tsx

```diff
@@ -0,0 +1,182 @@
+import cn from 'classnames';
+import React, {PureComponent} from 'react';
+import {Trans} from 'react-i18next';
+
+import Terminal from 'assets/images/terminalWindow.svg';
+import Flex from 'common/components/Flex/Flex';
+import styles from './mainSlide.scss';
+
+interface ISlide {
+	name: string;
+	title: any;
+	smallText: any;
+}
+
+interface IProps {
+	mainSlidesLength: number;
+	slide: ISlide;
+	currentSlideIndex: number;
+	changeSlide: any;
+	isAnimated: boolean;
+	order: number;
+	mainSlidesOrder: number[];
+	mainSlidesContent: ISlide[];
+}
+
+interface IState {
+	isMenuOpen: boolean;
+}
+
+class MainSlide extends PureComponent<IProps, IState> {
+	constructor(props: IProps) {
+		super(props);
+		this.state = {
+			isMenuOpen: false,
+		};
+	}
+
+	public componentDidUpdate(props: IProps) {
```

Comment:
unnecessary update after update. Remove this method

Reply:
done